src/main/scala/nl.knaw.dans.easy.deposit/servlets/DepositServlet.scala

```diff
@@ -36,16 +35,7 @@ class DepositServlet(app: EasyDepositApiApp)
   extends ProtectedServlet(app)
     with FileUploadSupport {
 
-  configureMultipartHandling(app.multipartConfig.copy(location = app.multipartConfig.location
```

Comment:
is this whole check no longer necessary?

Reply:
The app class does a same-mount check by creating and moving files. That makes these checks redundant.